modulus/utils/filesystem.py

```diff
@@ -132,6 +132,7 @@ def _download_cached(
 ) -> str:
     sha = hashlib.sha256(path.encode())
     filename = sha.hexdigest()
+    print("_download_cached, local_cache_path: ", local_cache_path)
```

Comment:
Same here, would be good to keep the print statements in the example level

Reply:
sorted